packages/dds/tree/src/test/shared-tree/undo.spec.ts

```diff
@@ -453,6 +464,137 @@ describe("Undo and redo", () => {
 		revertible.revert();
 		assert.equal(view.root.foo, 1);
 	});
+
+	it("reverts original & forked revertibles after making change to the original view", () => {
+		const originalView = asTreeViewAlpha(createLocalSharedTree("testSharedTree"));
+
+		const {
+			undoStack: undoStack1,
+			redoStack: redoStack1,
+			unsubscribe: unsubscribe1,
+		} = createTestUndoRedoStacks(originalView.events);
+
+		assert(originalView.root.child !== undefined);
+		originalView.root.child.propertyOne = 256; // 128 -> 256
+
+		const forkedView = originalView.fork();
+
+		const propertyOneUndo = undoStack1.pop();
+		const clonedPropertyOneUndo = propertyOneUndo?.clone(forkedView);
+
+		propertyOneUndo?.revert();
```

Comment:
we should prob have the same test where the forked revert happens first

Reply:
Added as `reverts cloned revertible prior to original revertible`.